src/constants/coreSettings.ts

```diff
@@ -744,5 +744,37 @@ export const CORE_SETTINGS: SettingParams[] = [
     type: 'boolean',
     defaultValue: true,
     versionAdded: '1.10.5'
+  },
+  {
+    id: 'Comfy.ModelLibrary.AllowedSources',
+    category: ['Comfy', 'Model', 'Download'],
+    name: 'Model Download Allowed Sources',
+    type: 'hidden',
```

Comment:
There does not seem to exists an array<string> type so relying on hidden for now. I don't think allowing configuration in Interface brings a lot of value